pyServer/manifests/windows/aks

```diff
@@ -5,6 +5,7 @@ copy,/Windows/System32/winevt/Logs/Application.evtx
 echo,## Kubelet and Kube proxy Logs ##
 copy,/k/*.log
 copy,/k/*.err
+copy,/k/config
```

Comment:
How big is the file roughly? Does it contains customer secret/credentials? is it a binary or text? We collect this information for performance tuning and information bookkeeping for long term